apps/prairielearn/src/lib/instructorFiles.ts

```diff
@@ -174,15 +178,28 @@ export function getPaths(
   const found = invalidRootPaths.find((invalidRootPath) => contains(invalidRootPath, workingPath));
   if (found) {
     throw new error.AugmentedError('Invalid working directory', {
-      info: html`
-        <p>The working directory</p>
-        <div class="container">
-          <pre class="bg-dark text-white rounded p-2">${workingPath}</pre>
-        </div>
-        <p>must <em>not</em> be inside the directory</p>
-        <div class="container"><pre class="bg-dark text-white rounded p-2">${found}</pre></div>
-        <p>when looking at <code>${locals.navPage}</code> files.</p>
-      `,
+      info:
+        found === workingPath
+          ? html`
+              <p>The working directory</p>
+              <div class="container">
+                <pre class="bg-dark text-white rounded p-2">${workingPath}</pre>
+              </div>
+              <p>
+                cannot be accessed directly when looking at <code>${locals.navPage}</code> files.
+              </p>
+            `
+          : html`
+              <p>The working directory</p>
+              <div class="container">
+                <pre class="bg-dark text-white rounded p-2">${workingPath}</pre>
+              </div>
+              <p>must <em>not</em> be inside the directory</p>
+              <div class="container">
+                <pre class="bg-dark text-white rounded p-2">${found}</pre>
+              </div>
+              <p>when looking at <code>${locals.navPage}</code> files.</p>
+            `,
```

Comment:
The error message when the found path matched the working path was confusing, so I added a separate case.